pygeoapi/flask_app.py

```diff
@@ -55,11 +56,42 @@
     from flask_cors import CORS
     CORS(APP)
 
-APP.config['JSONIFY_PRETTYPRINT_REGULAR'] = \
-    CONFIG['server'].get('pretty_print', True)
+APP.config['JSONIFY_PRETTYPRINT_REGULAR'] = CONFIG['server'].get(
+    'pretty_print', True)
 
 api_ = API(CONFIG)
 
+OGC_SCHEMAS_LOCATION = CONFIG['server'].get('ogc_schemas_location', None)
+
+if (OGC_SCHEMAS_LOCATION is not None and
+        not OGC_SCHEMAS_LOCATION.startswith('http')):
+    # serve the OGC schemas locally
+
+    @APP.route('/schemas/<path:path>', methods=['GET'])
+    def schemas(path):
+        """
+        Serve OGC schemas locally
+
+        :param path: path of the OGC schema document
+
+        :returns: HTTP response
+        """
+
+        mimetypes.add_type('text/plain', '.yaml')
+        mimetypes.add_type('text/plain', '.yml')
+
+        full_filepath = os.path.join(OGC_SCHEMAS_LOCATION, path)
```

Comment:
Should we catch an exception somewhere if the local file doesn't exist?

Reply:
Isn't this covered by starlette/flask accordingly?

Reply:
Yes, it is but does it make sense to give the users a `404` for a file that should have been placed manually by the admin? Maybe a fall back to the default?

Reply:
For `starlette_app.py` what do you think of this below?:

```python
if (OGC_SCHEMAS_LOCATION is not None and
        not OGC_SCHEMAS_LOCATION.startswith('http')):
    try:
        os.path.exists(OGC_SCHEMAS_LOCATION)
        # serve the OGC schemas locally
        app.mount('/schemas', StaticFiles(directory=OGC_SCHEMAS_LOCATION))
    except FileNotFoundError:
        raise RuntimeError(“Wrong local path location”)
```

Reply:
Good idea.  Updated.